Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -443,6 +443,8 @@ class BrowserViewController: UIViewController {
         // Re-show toolbar which might have been hidden during scrolling (prior to app moving into the background)
         scrollController.showToolbars(animated: false)
     }
+    
+    let downloader = NTPDownloader()
```

Comment:
This should follow the adblock downloading approach, and execute inside `willFinishLaunchingWithOptions` inside app delegate.